tasks/proposals/setup-emission-admin.ts

```diff
@@ -0,0 +1,86 @@
+import { loadPoolConfig } from "../../helpers/market-config-helpers";
```

Comment:
split it to reusable task and single use proposal, can be part of the 1st proposal that we will have related to incentives